atomic-design.md

```diff
@@ -0,0 +1,101 @@
+# Atomic Design Guide
+
+> Helpful guidance and tips for building front-end components to Atomically.
+
+## What is Atomic Design
+
+Created by [Brad Frost](http://bradfrost.com/blog/post/atomic-web-design/), Atomic Design is a system for building and maintaining reusable components (we refer to them as elements) for a web platform. By creating simple components, and combining them to build larger and more complex components, we can ensure a consistent design throughout our projects.
+
+## What's an atomic element?
+
+An atomic element is any self contained code pattern that follows the Atomic Design principles. The types of elements are atoms, molecules, organisms, and templates.
+
+### Atoms
+
+- The base unit
+- Think buttons, input fields and other html tags
+- Abstract and not too useful on their own
+
+### Molecules
+
+- It can do something
+- A series of atoms combined to serve a purpose
+
+### Organisms
+
+- Self contained chunks with different functionalities to form a distinct section
```

Comment:
I wasn't sure if section was right since some things like the expandables aren't whole sections.